### ISSUE TITLE
Unclear warning about "sort-imports" rule when mixing single and multiple imports in a single line

### ISSUE THREAD

**Issue by globexdesigns** [eslint/eslint#5305]
The following code:

```
import React, {Component} from 'react';
```

Returns the following error: `Member 'Component' of the import declaration should be sorted alphabetically`

There appears to be no way to control the case where you're importing a default as well as multiple parts of a module.

**Comment by ilyavolodin** [comment]
Do you still get the same error if you do it the other way around:

```js
import {Component}, React from 'react';
```

**Comment by wyze** [comment]
That is not valid syntax.

Gives `Unexpected token` from Babel. ESLint throws fatal error, `Parsing error: Unexpected token`.

**Comment by globexdesigns** [comment]
Correct. `import {Component}, React from 'react';` is not valid syntax and doesn't work.

**Comment by ilyavolodin** [comment]
Ah, sorry, not using es6 myself, so wasn't aware of that. In that case, it sounds like a bug. @eslint/eslint-team how should this be fixed?

**Comment by mysticatea** [comment]
I think the rule should check members inside of curly braces.
On ESTree, the rule should check only [ImportSpecifier](https://github.com/estree/estree/blob/master/es6.md#importspecifier)s. The `React` is a `ImportDefaultSpecifier`.

**Comment by globexdesigns** [comment]
So then, the sorting will ignore the ImportDefaultSpecifier? ie:

```
// No ESLint warning
import a from 'a';
import {b} from 'b';
import c, {d} from 'c';

// ESLint warning
import a from 'a';
import {c} from 'c';
import b, {d} from 'b';
```

**Comment by mysticatea** [comment]
Ah, I meant about sorting inside of one import declaration.

**Comment by mysticatea** [comment]
I'm working on this.
